src/ui/components/RecommendedBadge/styles.scss

```diff
@@ -0,0 +1,33 @@
+@import '~ui/css/styles';
+
+$recommended-text-color: #e25920;
+
+.RecommendedBadge {
+  align-items: center;
+  color: $recommended-text-color;
+  display: flex;
+  font-size: $font-size-m-smaller;
+  line-height: 1.2;
+  margin-bottom: 6px;
+
+  @include respond-to(large) {
+    justify-content: flex-end;
+  }
+
+  .Icon {
+    @include margin(0, 6px, 0, 0);
+  }
+
+  a,
+  a:link,
+  a:visited {
+    color: $recommended-text-color;
+    font-weight: normal;
+    text-decoration: none;
+  }
+
+  a:active,
+  a:hover {
+    color: $link-color;
```

Comment:
Honestly, I find this color a bit ugly when used next to the Trophy icon color:

![Screen Shot 2019-05-03 at 10 10 08](https://user-images.githubusercontent.com/217628/57126100-fe8c4c80-6d8b-11e9-9cdd-ef163924bfd6.png)

Reply:
It's true, we should probably make this a different shade of orange, so that it's obvious to a user that it's changing, but not so ugly/jarring. @brassy- could you please comment on this as well?